backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/window/model/DocumentCollection.java

```diff
@@ -790,10 +789,17 @@ private DocumentPath duplicateDocumentInTrx(final DocumentPath fromDocumentPath)
 
 		final TableRecordReference fromRecordRef = getTableRecordReference(fromDocumentPath);
 
+		return duplicateDocumentInTrxGeneric(fromRecordRef, fromDocumentPath.getAdWindowIdOrNull());
+
+	}
+
+	public DocumentPath duplicateDocumentInTrxGeneric(final TableRecordReference fromRecordRef, final AdWindowId windowId)
```

Comment:
if they can't/shall not be null, then please use lombok's `@NonNull` annotation

Reply:
Addrssed this